src/mixins/filing-template-mixin.ts

```diff
@@ -586,6 +587,24 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     return filing
   }
 
+  /** The list of completing parties. */
+  // OrgPersonIF[]
+  completingParties (orgPersonList): OrgPersonIF[] {
+    const completingPartiesList = orgPersonList.filter(party =>
```

Comment:
There should be just 1 completing party, so you can use `find` instead of `filter`.

Reply:
Obsolete code. Resolving.